src/dataflow/core/lispress.py

```diff
@@ -313,51 +277,11 @@ def _desugar_gets(sexp: Lispress) -> Lispress:
             return [
                 DataflowFn.Get.value,
                 _desugar_gets(obj),
-                OpType.Value.value,
-                ["Path", f'"{key}"'],
+                [OpType.Value.value, ["Path", f'"{key}"']],
```

Comment:
Reader macros are now `(# (type value))` instead the previous behavior where they were flattened into the parent.

Reply:
That happens inside sexp_to_str (and also in _render_lines in #37).

Reply:
ah, so this simplifies the code without changing that? beautiful